airbyte-integrations/connectors/source-tiktok-marketing/source_tiktok_marketing/streams.py

```diff
@@ -657,7 +660,7 @@ def stream_slices(self, stream_state: Mapping[str, Any] = None, **kwargs) -> Ite
                 yield slice
 
     def path(self, *args, **kwargs) -> str:
-        return "reports/integrated/get/"
+        return "report/integrated/get/"
```

Comment:
@grubberr can you confirm this change is right? Asking because it's not mentioned in the PR description, not because it looks problematic

Reply:
@girarda Let me answer your question since we're working together with @grubberr on these improvements. it should be correct. I've just tested it. @grubberr correct me, if there is more to add.